.github/workflows/integration-tests.yml

```diff
@@ -1,16 +1,12 @@
 name: Integration Tests
 
 on:
-  workflow_dispatch:
+  workflow_call:
     inputs:
       environment:
-        type: choice
+        type: string
         description: 'environment to run test in'
         required: true
-        options:
-          - staging
```

Comment:
I thought this was used by the GitHub action UI to show the possible values, to see what I mean go to https://github.com/DataBiosphere/terra-data-catalog/actions/workflows/integration-tests.yml and select Run Workflow

Reply:
Choice seemed to not be an option for workflow_call - maybe I was misunderstanding the API docs though and there's another way to specify it?